definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+ Definition of done
+
+This document clarifies, when an product backlog item is ready. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api (diagrams?)
+* [ ] Tutorials
```

Comment:
In my opinion, to unspecific, especially for new people. I suggest to at least add a link to https://wiki.ros.org/pilz_robots/Tutorials/ and https://github.com/PilzDE/pilz_application_templates.

In general, I suggest that we come up with a short overall user documentation & tutorial concept which can be linked here.

The overall user documentation & tutorial concept should explain:
- where new user documentations & tutorials need be added and
- what needs to be covered on the different sites:
  - https://wiki.ros.org/pilz_robots/Tutorials/
  - https://github.com/PilzDE/pilz_application_templates
  - https://github.com/PilzDE/pilz_industrial_motion/tree/melodic-devel/pilz_robot_programming/examples
  - What about our PythonAPI documentation which is currently only completely available  (if I'm correct) if I execute `rosdoc_lite`? See in [pilz_robot_programming/Readme.rst](https://github.com/PilzDE/pilz_industrial_motion/blob/melodic-devel/pilz_robot_programming/Readme.rst)
  - What do we explain in the different README.md files in the different packages like [pilz_robots/README.md](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md)?

Furthermore: We "only" cover our PythonAPI, right now. But what about _our_ ROS-API which is build up by our ROS messages? Do we want to document these things in more detail in the future? If so, where?

Reply:
I support this idea.

> We "only" cover our PythonAPI, right now.

What do you mean? The whole ROS-API is available on https://wiki.ros.org, e.g.
http://docs.ros.org/melodic/api/pilz_robot_programming/html/
http://docs.ros.org/melodic/api/prbt_hardware_support/html/prbt_hardware_support/

Reply:
> The whole ROS-API is available on

Ah, I didn't find these sites :)

Nevertheless, how do all these sites fit into a coherent well structured overall product documentation? Or differently phrased: What info should be written down where? @jschleicher  What do you think?